src/Pelagos/Bundle/AppBundle/Resources/views/DataDiscovery/template-dataset-row.html.twig

```diff
@@ -1,13 +1,23 @@
 <table hidden>
 <tr id="template-data-row-available" hidden>
     <td class="info">
-        <span id="container-dataset-authors"><span id="dataset-authors"></span>. </span>
-        <span id="container-dataset-year"><span id="dataset-year"></span>. </span>
-        <a href="{{ path('pelagos_app_ui_dataland_default', {'udi' : 'placeholder-udi'}) }}" target="_blank" id="dataset-title">
-        </a>.
-        <span id="container-dataset-research-group"><span id="dataset-research-group"></span>. </span>
-        <span id="container-dataset-doi"><span id="label-dataset-doi">DOI: </span><a href="http://dx.doi.org/{{ 'placeholder-doi' }}" target="_blank" id="dataset-doi"></a>. </span>
-        <span id="container-dataset-udi"><span id="label-dataset-udi">UDI: <a href="{{ path('pelagos_app_ui_dataland_default', {'udi' : 'placeholder-udi'}) }}" target="_blank" id="dataset-udi"></a>. </span></span>
+        <span id="container-dataset-authors">
```

Comment:
Probably we can have the anchor tag for the whole row?

Reply:
These rows are dynamically generated by the addRows() method of data-discovery.js based on some json data. There's no anchor tag at all, just a clink event bound to the row.